src/main/java/com/hackathon03/codeguru/utils/SQLInjection.java

```diff
@@ -0,0 +1,35 @@
+package com.hackathon03.codeguru.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+public class SQLInjection {
+
+    public void executeSqlStatementNoncompliant(HttpServletRequest request, java.sql.Connection connection) {
+        final String favoriteColor = request.getParameter("favoriteColor");
+        try {
+            String sql = "SELECT * FROM people WHERE favorite_color='" + favoriteColor + "'";
+            java.sql.Statement statement = connection.createStatement();
+            // Noncompliant: user-given input is not sanitized before use.
+            statement.execute(sql);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

We detected an SQL command that might use unsanitized input. This can result in an SQL injection. To increase the security of your code, sanitize inputs before using them to form a query string.

[Learn more](https://cwe.mitre.org/data/definitions/89.html)